bases/renku_data_services/background_jobs/main.py

```diff
@@ -0,0 +1,43 @@
+"""Script to run a variety of background jobs independently from the data services deployment."""
+
+import argparse
+import asyncio
+import logging
+
+from renku_data_services.authz.admin_sync import sync_admins_from_keycloak
+from renku_data_services.authz.authz import Authz
+from renku_data_services.background_jobs.config import SyncConfig
+from renku_data_services.background_jobs.utils import bootstrap_user_namespaces
+from renku_data_services.migrations.core import run_migrations_for_app
+
+logging.basicConfig(level=logging.INFO)
+
+
+async def main() -> None:
+    """Synchronize data from Keycloak and the user database."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "tasks",
+        nargs="+",
+        choices=["keycloak_events_sync", "keycloak_users_sync", "keycloak_admins_sync", "user_namespaces_bootstrap"],
+    )
```

Comment:
I think we should use subparsers for different actions. That allows to have different arguments if we ever need it, is also exhaustive and automatically handles errors. It also means we don't need a match statement but can just have it be functions that get called directly.